src/views/homepage/components/chiclets/chiclets.module.css

```diff
@@ -4,30 +4,47 @@
  */
 
 .root {
-	margin-top: 72px;
+	margin-top: 122px;
+
+	@media only screen and (max-width: 1440px) {
+		margin-top: 72px;
+	}
+
+	@media (--dev-dot-mobile) {
+		margin-top: 32px;
+	}
 }
 
 .label {
-	/* composition */
-	composes: hds-typography-display-100 from global;
-
-	/* properties */
+	composes: hds-font-weight-bold from global;
 	color: var(--token-color-foreground-high-contrast);
-	font-weight: var(--token-typography-font-weight-medium);
-}
+	font-family: var(--token-typography-display-400-font-family);
+	font-size: var(--token-typography-display-400-font-size);
+	line-height: var(--token-typography-display-400-line-height);
+	margin: 0;
+	margin-bottom: 24px;
+	padding: 0;
 
-.nav {
-	margin-top: 30px;
+	@media (--dev-dot-mobile) {
+		font-family: var(--token-typography-display-300-font-family);
+		font-size: var(--token-typography-display-300-font-size);
+		line-height: var(--token-typography-display-300-line-height);
+		margin-bottom: 20px;
+	}
 }
 
 .navList {
 	display: flex;
 	flex-wrap: wrap;
-	gap: 20px;
+	gap: 16px;
 	list-style: none;
 	margin: 0;
 	padding: 0;
 	width: 100%;
+
+	@media (--dev-dot-tablet-down) {
+		gap: 8px;
```

Comment:
⛏️ Similar to the desktop wrap issue, we might need to bump this down to `7px` or something (or maybe some other change?) if we want the chiclets to stay on one line at specific viewport widths (`999px` shown below):

![CleanShot 2023-04-20 at 10 38 51@2x](https://user-images.githubusercontent.com/4624598/233401122-c5f96050-fa9d-440d-9bac-f13e33ae5a98.png)